cli.js

```diff
@@ -1,103 +1,53 @@
 #!/usr/bin/env node
 // Simple CLI for KaTeX.
 // Reads TeX from stdin, outputs HTML to stdout.
+// To run this from the repository, you must first build KaTeX by running
+// `npm install` and `npm run dist`.
+
 /* eslint no-console:0 */
 
 const katex = require("./");
+const json = require("./package.json");
```

Comment:
I think the name `package` or using object destructuring, `{version}` is better.